src/language/nmodl.yaml

```diff
@@ -438,6 +438,18 @@
                               is base class and defines common interface for these nodes.
 
                       children:
+                        - InstanceStruct:
+                            nmodl: "INSTANCE_STRUCT "
```

Comment:
I wanted to rename this but we will do it in separate PR where will rename it in C backend as well.